tuner/generate_tuning_config.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Given an input tuning spec, generates the config files used by the tuner.
+
+import argparse
+
+
+def main() -> int:
```

Comment:
We should also drop this file until it's used by libtuner.py

Reply:
removed